challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/while/exercise_01.js

```diff
@@ -0,0 +1,36 @@
+/*
+    1. Create a while loop that will print out all the even numbers between 10 and 40.
+
+    2. Create a while loop that will print out all the odd numbers between 10 and 40.
+*/
+
+print("Select an option: ");
+print("1. Even numbers between 10 and 40");
+print("2. Odd numbers between 10 and 40");
+
+switch (prompt("Enter your choice: ")) {
+  case "1":
+    evenNumbers();
+    break;
+  case "2":
+    oddNumbers();
+    break;
+}
+
+//even numbers
+let i = 10;
```

Comment:
You should take in account what `i` variable is storing. Is this variable referring to any index in this case?

⚠️ FIX: rename properly `i` variable